### PR TITLE
PMM-9562-PMM-server-IP-warning-grafana8

### PR DESCRIPTION
**Pull request by solovevayaroslavna**
**What this PR does / why we need it**: Added a notification to the AddKubernetesModal modal form.

**Which issue(s) this PR fixes**: https://jira.percona.com/browse/PMM-9562

### CODE REVIEW

File: public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerURLWarning.utils.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  <>
+    {Messages.dbcluster.publicAddressWarningBegin}
+    &nbsp;
+    <a href={ADVANCED_SETTINGS_URL} className={className}>
+      {Messages.dbcluster.publicAddressWarningLink}
+    </a>
+    &nbsp;
+    {Messages.dbcluster.publicAddressWarningEnd}
+  </>
+);
```
Comment (fabio-silva) [inline public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerURLWarning.utils.tsx:15]: Couldn't this be a functional component instead?

File: public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      } finally {
+        setSettingsLoading(false);
+      }
+    };
+
+    getSettings();
+  }, []);
+
+  return showMonitoringWarning;
+};
```
Comment (fabio-silva) [inline public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts:29]: Warning: this is gonna be deeply affected by this PR: https://github.com/percona-platform/grafana/pull/340
We'll no longer call the API, to get the settings, but rely on the store instead. This said, I think this hook won't be necessary.

**Review by fabio-silva — CHANGES_REQUESTED** [review]

File: public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts (hunk 2 above)
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts:29]: thanks, I understand, I'm going to study this PR

**Comment by fabio-silva** [comment]
@solovevayaroslavna please update the branch

File: public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts (hunk 2 above)
Comment (solovevayaroslavna) [reply public/app/percona/dbaas/components/PMMServerURLWarning/PMMServerUrlWarning.hooks.ts]: fixed

**Review by fabio-silva — APPROVED** [review]

**Pull request merged**